ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: Apache 2
 
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.19;
```

Comment:
note: This means integrators will need solidity 0.8.19 right

Reply:
I guess they won't as long as we seperate out the externally facing stuff into its own file with solidity 0.8.0

Reply:
>create 2 copies of the public IWormholeRelayer.sol interface: 1 using UDVTs and the other without. Generate the untyped version by programmatically find-and-replace UDVTs for the underlying types.

Yes for the typed version, will create an untyped version in a flup pr

Reply:
not if we have the dual approach that Joe mentioned where we replace the custom types with their underlying datatype (using a script so we don't have to maintain this manually).

Reply:
Note that this pr has the worst of both worlds:
- no udvts on the public api itself, 
- yet the solidity interface file requires 0.8.19 
Please kindly ignore this for now :p